src/include/daos_pool.h

```diff
@@ -9,6 +9,8 @@
 #ifndef __DAOS_POOL_H__
 #define __DAOS_POOL_H__
 
+#define daos_pool_connect daos_pool_connect2
```

Comment:
Hmm, do we still need this?

Reply:
Sorry. Just found your comment. Johann suggested to keep this macro to use daos_pool_connect2() in the API provided by libdaos.so. In C/C++, we use daos_pool_connect(). We have to use daos_pool_connect2() since it is the only function name exposed.